backend/src-common/src/main/java/com/siemens/sw360/components/summary/ProjectSummary.java

```diff
@@ -54,4 +57,32 @@ protected static void setSummaryFields(Project document, Project copy) {
             }
         }
     }
+
+    protected static void setDefaultFields(Project document, Project copy) {
+        copyField(document, copy, _Fields.ID);
```

Comment:
Only the name is a required field according to the thrift struct. ID, NAME and VERSION are set in the "unaccessible..." and "accessible..." summary, because they have to be displayed (or used) in the linked project view. In principle that could change again and is independent of other summaries. And in the SUMMARY, I set all fields anyway by iterating over all fields - there would be no advantage in setting some minimal fields already before. And in all other cases (Summaries SHORT, EXPORT, DETAILED_EXPORT, HOME - no idea if they're all used, probably not), also the description and clearing team are set at the moment. So, I would prefer to leave it as it is and not to split the default case in a minimal case and some other fields needed for the default. - OK?